### PR TITLE
Update swift stack to Swift 5.1, add debugging support

### PR DESCRIPTION
**Pull request by djones6**
Updates the `incubator/swift` stack to use Swift 5.1, and adds debugging support.

### Checklist:

- [x] Read the [Code of Conduct](https://github.com/appsody/website/blob/master/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md).

- [ ] Followed the [commit message guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).

- [x] Stack adheres to [Appsody stack structure](https://github.com/appsody/website/blob/master/content/docs/stacks/stack-structure.md).

### Modifying an existing stack:

- [x] Updated the stack version in `stack.yaml`

### CODE REVIEW

File: incubator/swift/image/Dockerfile-stack [hunk 1]
```diff
@@ -1,9 +1,4 @@
-FROM swift:5.0
-
-RUN apt-get update \
- && apt-get install -y zlib1g-dev libcurl4-openssl-dev libssl-dev \
- && apt-get clean \
- && echo 'Finished installing dependencies'
+FROM swift:5.1
```
Comment (ianpartridge) [inline incubator/swift/image/Dockerfile-stack]: We should leave `zlib1g-dev` in for now as I think Chris's blog post needs it for the Vapor demo.

Comment (ianpartridge) [reply incubator/swift/image/Dockerfile-stack]: https://medium.com/appsody/server-side-swift-a77bff1d8f1

File: incubator/swift/stack.yaml [hunk 2]
```diff
@@ -1,10 +1,13 @@
 name: Swift
-version: 0.1.4
+version: 0.2
```
Comment (skoh7645) [inline incubator/swift/stack.yaml]: 0.2 --> 0.2.0 for format consistency

File: incubator/swift/image/Dockerfile-stack [hunk 3]
```diff
@@ -18,9 +13,12 @@ ENV APPSODY_RUN="swift run"
 ENV APPSODY_RUN_ON_CHANGE="swift run"
 ENV APPSODY_RUN_KILL=true
 
-ENV APPSODY_DEBUG=""
-ENV APPSODY_DEBUG_ON_CHANGE=""
-ENV APPSODY_DEBUG_KILL=true
+# Allow remote debugging. The 'appsody run' command must include the following
```
Comment (skoh7645) [inline incubator/swift/image/Dockerfile-stack]: `appsody run` -->`appsody debug`

File: incubator/swift/image/Dockerfile-stack (hunk 1 above)
Comment (djones6) [reply incubator/swift/image/Dockerfile-stack]: Could the blog post use the `kitura` stack instead for the Kitura bit? That would seem more appropriate.

We can certainly add zlib back in again - turns out things like `async-http-client` now require that too.

Comment (ianpartridge) [reply incubator/swift/image/Dockerfile-stack]: Yes, leaving zlib in here and updating the Kitura bit of the blog post is my preference.

**Comment by djones6** [comment]
@skoh7645 @ianpartridge Changes made.

I've tested the debug configuration locally - you can spin up vscode, CMD-Shift-B and run 'appsody debug', then launch the debug configuration.

Upon changing the app, eg. "Hello World" -> "Hello Debug", the debug container will rebuild the image automatically, and you can then launch the debug configuration again and see the new output.

File: incubator/swift/image/Dockerfile-stack (hunk 1 above)
Comment (djones6) [reply incubator/swift/image/Dockerfile-stack]: We've decided to leave those dependencies in the swift stack for now, and a better long-term solution would be to discover them from the dependency tree.  Can discuss in https://github.com/appsody/stacks/issues/432

**Pull request merged**